README.md

```diff
@@ -59,56 +64,45 @@ git clone https://github.com/Qiskit/retworkx.git
 
 retworkx uses [PyO3](https://github.com/pyo3/pyo3) and
 [setuptools-rust](https://github.com/PyO3/setuptools-rust) to build the
-python interface. Unfortunately, this means you need to use nightly rust
-because PyO3 only works with nightly at this point. You can use
-[rustup](https://rustup.rs/) to install rust nightly.
-
-Once you have nightly rust and cargo installed you can easily install
-retworkx into your python environment using pip. Once you have a local
-clone of the repo, change your current working directory to the root of
-the repo. To set the compiler for `retworkx` to nightly rust using
-`rustup`, run the following from this directory:
+python interface. Which enables using standard python tooling to work. So,
```

Comment:
-> "python interface which enables the use of standard python tooling"

Reply:
Done in https://github.com/Qiskit/retworkx/pull/91/commits/504ab0fc206d8e6c7c27474f89c223e00b139969